### PR TITLE
Div, mult and more features

### PR DESCRIPTION
**Pull request by lasseebert**
Adds extra features like division and multiplication. One commit per feature :)

- `div` and `mult`
- `zero?`, `positive?` and `negative?`
- `round`

### CODE REVIEW

File: lib/quantity.ex [hunk 1]
```diff
@@ -46,19 +53,38 @@ defmodule Quantity do
 
   @doc """
   Parses a string representation of a quantity (perhaps generated with to_string/1)
+
+  iex> Quantity.parse("99.0 red_balloons")
+  {:ok, Quantity.new(~d[99.0], "red_balloons")}
+
+  iex> Quantity.parse("15 bananas/monkey")
+  {:ok, Quantity.new(~d[15], {:div, "bananas", "monkey"})}
```
Comment (skovmand) [inline lib/quantity.ex]: `bananas/monkey` :smile:

File: lib/quantity/math.ex [hunk 2]
```diff
@@ -154,4 +154,70 @@ defmodule Quantity.Math do
...
+
+  @doc """
+  Divide a Quantity by a scalar or another Quantity
+
+  iex> Quantity.div(~Q[15 $], ~Q[10 banana])
+  ~Q[1.5 $/banana]
+  iex> Quantity.div(~Q[15 $], ~d[10])
+  ~Q[1.5 $]
+  iex> Quantity.div(~Q[15 $], ~Q[10 $])
+  ~d[1.5]
```
Comment (skovmand) [inline lib/quantity/math.ex]: Nice. Perhaps add an empty line between tests for consistency with other doctests?

File: lib/quantity/sigils.ex [hunk 3]
```diff
@@ -19,6 +19,8 @@ defmodule Quantity.Sigils do
 
   iex> ~Q[12.340 kwh]
   Quantity.new(~d[12.340], "kwh")
+  iex> ~Q[15 $/banana]
+  Quantity.new(~d[15], {:div, "$", "banana"})
```
Comment (skovmand) [inline lib/quantity/sigils.ex]: Would also be good with an empty line between doctests here.

File: README.md [hunk 4]
```diff
@@ -102,6 +102,27 @@ sales_2019 |> Quantity.sum(-2, "EUR")
...
+
+```elixir
+cost = ~Q[20.00 $]
+amount = ~Q[50 banana]
+cost_per_banana = Quantity.div(cost, amount)
+# ~Q[0.40 $/banana]
+
+new_amount = ~Q[2000 banana]
+Quantity.mult(cost_per_banana, new_amount)
+# ~Q[800.00 $]
```
Comment (skovmand) [inline README.md]: I think this example could be added to the top brief examples in README.md too

File: lib/quantity/math.ex (hunk 2 above)
Comment (lasseebert) [reply lib/quantity/math.ex]: Consistency is good :+1: 
Added in 5c5d447e2fdf85f82d30b107a783a40543047126

File: lib/quantity/sigils.ex (hunk 3 above)
Comment (lasseebert) [reply lib/quantity/sigils.ex]: Also added in 5c5d447e2fdf85f82d30b107a783a40543047126 (and also added a sigil test with `:mult`)

File: README.md (hunk 4 above)
Comment (lasseebert) [reply README.md]: Agreed force-pushed the readme commit (cf49e104475328619fd31b8ef60a1d6cfa0f29b2) with that change.

File: lib/quantity/sigils.ex (hunk 3 above)
Comment (skovmand) [reply lib/quantity/sigils.ex]: Nice! :+1:

File: lib/quantity/math.ex (hunk 2 above)
Comment (skovmand) [reply lib/quantity/math.ex]: Super. Looks good.

**Comment by skovmand** [comment]
Merge at will :-)

**Comment by skovmand** [comment]
Actually, one more thing @lasseebert .... I just saw the README.md file in its entirety and I think it would be nice with two examples for `parse/1` showing multiplication and division. Perhaps the same too for the `~Q` examples?

**Comment by lasseebert** [comment]
@skovmand: Added some README in https://github.com/wise-home/quantity/pull/8/commits/43d75284310590553290977b06a4776aad56ddef

**Comment by skovmand** [comment]
@lasseebert Very nice, just what I had in mind.

**Pull request merged**